<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartupDialog</class>
 <widget class="QDialog" name="StartupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>プロファイル</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ProfileComboBox" name="profileComboBox"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>設定名</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>有効化</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="validCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>日時</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="7" colspan="5">
       <widget class="QDateTimeEdit" name="scheduleDateTimeEdit">
        <property name="displayFormat">
         <string>yyyy/MM/dd HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QCheckBox" name="ThuCheckBox">
        <property name="text">
         <string>木</string>
        </property>
       </widget>
      </item>
      <item row="2" column="10">
       <widget class="QSpinBox" name="weeksSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QCheckBox" name="FriCheckBox">
        <property name="text">
         <string>金</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>予約時間</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QCheckBox" name="WedCheckBox">
        <property name="text">
         <string>水</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="6">
       <widget class="QLineEdit" name="secondsLineEdit"/>
      </item>
      <item row="3" column="9" colspan="3">
       <widget class="QComboBox" name="secondsComboBox">
        <item>
         <property name="text">
          <string>秒ごと</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>分ごと</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>時間ごと</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="8" colspan="2">
       <widget class="QCheckBox" name="SunCheckBox">
        <property name="text">
         <string>日</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="TueCheckBox">
        <property name="text">
         <string>火</string>
        </property>
       </widget>
      </item>
      <item row="2" column="11">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>週間毎</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9" colspan="3">
       <widget class="QTimeEdit" name="scheduleTimeEdit">
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QCheckBox" name="SatCheckBox">
        <property name="text">
         <string>土</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="MonCheckBox">
        <property name="text">
         <string>月</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QRadioButton" name="secondsRadioButton">
        <property name="text">
         <string>秒間スケジュール</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">radioButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QRadioButton" name="weeksRadioButton">
        <property name="text">
         <string>週間スケジュール</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">radioButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QRadioButton" name="oneshotRadioButton">
        <property name="text">
         <string>1回のみの実行</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">radioButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="everyDayCheckBox">
        <property name="text">
         <string>毎日</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>oneshotRadioButton</zorder>
     <zorder>scheduleDateTimeEdit</zorder>
     <zorder>weeksRadioButton</zorder>
     <zorder>label_3</zorder>
     <zorder>scheduleTimeEdit</zorder>
     <zorder>secondsRadioButton</zorder>
     <zorder>secondsComboBox</zorder>
     <zorder>TueCheckBox</zorder>
     <zorder>WedCheckBox</zorder>
     <zorder>ThuCheckBox</zorder>
     <zorder>FriCheckBox</zorder>
     <zorder>SatCheckBox</zorder>
     <zorder>SunCheckBox</zorder>
     <zorder>weeksSpinBox</zorder>
     <zorder>label_5</zorder>
     <zorder>secondsLineEdit</zorder>
     <zorder>MonCheckBox</zorder>
     <zorder>everyDayCheckBox</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProfileComboBox</class>
   <extends>QComboBox</extends>
   <header>profilecombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>profileComboBox</tabstop>
  <tabstop>validCheckBox</tabstop>
  <tabstop>oneshotRadioButton</tabstop>
  <tabstop>scheduleDateTimeEdit</tabstop>
  <tabstop>weeksRadioButton</tabstop>
  <tabstop>scheduleTimeEdit</tabstop>
  <tabstop>MonCheckBox</tabstop>
  <tabstop>TueCheckBox</tabstop>
  <tabstop>WedCheckBox</tabstop>
  <tabstop>ThuCheckBox</tabstop>
  <tabstop>FriCheckBox</tabstop>
  <tabstop>SatCheckBox</tabstop>
  <tabstop>SunCheckBox</tabstop>
  <tabstop>weeksSpinBox</tabstop>
  <tabstop>secondsRadioButton</tabstop>
  <tabstop>secondsLineEdit</tabstop>
  <tabstop>secondsComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StartupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StartupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="radioButtonGroup"/>
 </buttongroups>
</ui>
